include/position_util.h

```diff
@@ -3,7 +3,7 @@
 
 #include "position.h"
 
-s32 CalculateFacingDir(struct Position *originPos, struct Position *targetPos);
-s32 GetDistance(struct Position *pos1, struct Position *pos2);
+s32 GetDirectionTowardsPosition(struct Position *originPos, struct Position *targetPos);
+s32 GetChebyshevDistance(struct Position *pos1, struct Position *pos2);
```

Comment:
Do we have to put the formula name here? Or can we just put a note saying it's using that certain formula..

Reply:
I guess it's not necessary if there are no other distance calculation methods used. I can change this back to GetDistance().